### ISSUE TITLE
data-match-height for tabs not working in foundation 6.3

### ISSUE THREAD

**Issue by neerajmourya** [zurb/foundation-sites#9677]
data-match-height="true" option for tabs not working.

#### How to reproduce this bug:
Following is my test html:
```
<div class="row">
      <div class="large-12 columns">
        <ul class="tabs" data-active-collapse="true" data-match-height="true" data-tabs id="collapsing-tabs">
	  <li class="tabs-title is-active"><a href="#panel1c" aria-selected="true">Tab 1</a></li>
	  <li class="tabs-title"><a href="#panel2c">Tab 2</a></li>
	  <li class="tabs-title"><a href="#panel3c">Tab 3</a></li>
	  <li class="tabs-title"><a href="#panel4c">Tab 4</a></li>
	</ul>

	<div class="tabs-content" data-tabs-content="collapsing-tabs">
	  <div class="tabs-panel is-active" id="panel1c">
	    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
	  </div>
	  <div class="tabs-panel" id="panel2c">
	    <p>Vivamus hendrerit arcu sed erat molestie vehicula. Sed auctor neque eu tellus rhoncus ut eleifend nibh porttitor. Ut in nulla enim. Phasellus molestie magna non est bibendum non venenatis nisl tempor. Suspendisse dictum feugiat nisl ut dapibus.</p>
	  </div>
	  <div class="tabs-panel" id="panel3c">
	    <img class="thumbnail" src="assets/img/generic/rectangle-3.jpg">
	  </div>
	  <div class="tabs-panel" id="panel4c">
	    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
	  </div>
	</div>
      </div>
    </div>
```

#### What should happen:
All Tab contents should have equal heights.

#### What happened instead:
Following is the error i am getting on firefox console:
```
TypeError: this.options is undefined
Stack trace:
_setHeight/<@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/foundation.js:5856:15
.each@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/jquery.js:365:10
jQuery.prototype.each@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/jquery.js:137:10
_setHeight@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/foundation.js:5854:9
singleImageLoaded@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/foundation.js:1851:9
onImagesLoaded/<@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/foundation.js:1835:9
.each@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/jquery.js:365:10
jQuery.prototype.each@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/jquery.js:137:10
onImagesLoaded@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/foundation.js:1832:5
_init@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/foundation.js:5626:13
Tabs@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/foundation.js:5535:7
Foundation.reflow/</<@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/foundation.js:193:34
.each@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/jquery.js:365:10
jQuery.prototype.each@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/jquery.js:137:10
Foundation.reflow/<@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/foundation.js:175:9
.each@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/jquery.js:365:10
Foundation.reflow@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/foundation.js:167:7
foundation@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/vendor/foundation.js:275:7
@file:///home/neeraj/Downloads/foundation-6.3.0-custom/js/app.js:1:1
```
Following is the error i am getting on chrome:
```
foundation.js:195 TypeError: Cannot read property 'panelActiveClass' of undefined
    at HTMLDivElement.<anonymous> (foundation.js:5856)
    at Function.each (jquery.js:365)
    at jQuery.fn.init.each (jquery.js:137)
    at Tabs._setHeight (foundation.js:5854)
    at singleImageLoaded (foundation.js:1851)
    at HTMLImageElement.<anonymous> (foundation.js:1835)
    at Function.each (jquery.js:365)
    at jQuery.fn.init.each (jquery.js:137)
    at Object.onImagesLoaded (foundation.js:1832)
    at Tabs._init (foundation.js:5626)
(anonymous) @ foundation.js:195
```

**Comment by ahebrank** [comment]
See possible solution in #9678

**Comment by kball** [comment]
Fixed in #9685
